prototype/mc_shortest_path/arduino_mc_shortest_path/arduino_mc_shortest_path.ino

```diff
@@ -0,0 +1,148 @@
+#define mc_real float
```

Comment:
You shouldn't need to define this variable as it is already defined in the .h file ?